src/ProjectNeon/Assets/Features/Effects/AnyTargetHealthBelowThreshold.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using UnityEngine;
+
+public sealed class AnyTargetHealthBelowThreshold : ConditionalEffect
+{
+    private float _threshold;
+
+    public AnyTargetHealthBelowThreshold (Effect effect, float threshold) : base(effect)
+    {
+        _threshold = threshold;
+    }
+
+    public override bool Condition()
+    {
+        return Array.Exists(_target.Members, element => element.State[TemporalStatType.HP] < element.State.MaxHP() * _threshold );
```

Comment:
It's generally better to use `System.Linq`'s `.Any()` extension method for this. 

Also, for short lambdas, it's nicer to use a single-letter variable name such as `x` or `e`.